seisbench/data/geofon.py

```diff
@@ -343,3 +453,13 @@ def find(self, trace_id):
                 return self.short_dict[netsta]
             else:
                 return np.nan, np.nan, np.nan, np.nan
+
+
+class GEOFONv2(GEOFON):
```

Comment:
Before merging this into `main`, we should discuss if/when we will make the underlying dataset publicly available. In general, I would prefer not to put any dataset into `main` that are only private at GEOFON.